ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/composables/PaywallBackground.kt

```diff
@@ -1,34 +1,51 @@
 package com.revenuecat.purchases.ui.revenuecatui.composables
 
+import androidx.compose.foundation.Image
 import androidx.compose.foundation.layout.BoxScope
 import androidx.compose.runtime.Composable
 import androidx.compose.ui.Modifier
 import androidx.compose.ui.draw.BlurredEdgeTreatment
 import androidx.compose.ui.draw.alpha
 import androidx.compose.ui.draw.blur
 import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.res.painterResource
 import androidx.compose.ui.unit.dp
+import com.revenuecat.purchases.paywalls.PaywallData
+import com.revenuecat.purchases.ui.revenuecatui.R
 import com.revenuecat.purchases.ui.revenuecatui.data.processed.TemplateConfiguration
 import com.revenuecat.purchases.ui.revenuecatui.extensions.conditional
+import com.revenuecat.purchases.ui.revenuecatui.extensions.defaultBackgroundPlaceholder
 
 @Composable
 internal fun BoxScope.PaywallBackground(templateConfiguration: TemplateConfiguration) {
-    templateConfiguration.images.backgroundUri?.let {
-        RemoteImage(
-            urlString = it.toString(),
-            modifier = Modifier
-                .matchParentSize()
-                .conditional(templateConfiguration.configuration.blurredBackgroundImage) {
-                    // TODO-PAYWALLS: backwards compatibility for blurring
-                    blur(BackgroundUIConstants.blurSize, edgeTreatment = BlurredEdgeTreatment.Unbounded)
-                        .alpha(BackgroundUIConstants.blurAlpha)
-                },
-            contentScale = ContentScale.Crop,
+    val modifier = Modifier
+        .matchParentSize()
+        .conditional(templateConfiguration.configuration.blurredBackgroundImage) {
+            // TODO-PAYWALLS: backwards compatibility for blurring
+            blur(BackgroundUIConstants.blurSize, edgeTreatment = BlurredEdgeTreatment.Unbounded)
+                .alpha(BackgroundUIConstants.blurAlpha)
+        }
+
+    if (templateConfiguration.configuration.images.background == PaywallData.defaultBackgroundPlaceholder) {
+        Image(
+            modifier = modifier,
+            painter = painterResource(id = R.drawable.default_background),
```

Comment:
Something I talked with @tonidero is what if we had a generated background instead of a this default jpg with mountains and a lake. We could maybe do a gradient from background to foreground or something that doesn't' take space in the SDK and works fine in all device dimensions.

Not really feedback for this PR, just something I wanted to write somewhere lol

Reply:
> works fine in all device dimensions.

Well the image is blurred so it doesn't matter if it's squashed in different aspect ratios. But yeah if we can make something that looks nice to replace it I'm all for it.